barretenberg/sol/src/honk/BaseHonkVerifier.sol

```diff
@@ -23,6 +23,8 @@ import {Transcript, TranscriptLib} from "./Transcript.sol";
 
 import {RelationsLib} from "./Relations.sol";
 
+import {CommitmentSchemeLib as PCS, ShpleminiIntermediates} from "./CommitmentScheme.sol";
```

Comment:
If the ShpleminiIntermediates is moved within the library, then we can just import the one. 

It would be accessed via PCS.ShpleminiIntermediates